.agents/career-ops-pdf.md

```
@@ -0,0 +1,9 @@
+---
+description: Generate ATS-optimized CV PDF
+---
+
+Generate ATS-optimized CV using career-ops pdf mode.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)